docs/kpod-stats.1.md

```diff
@@ -0,0 +1,37 @@
+% kpod(1) kpod-stats - Display a live stream of 1 or more containers' resource usage statistics
+% Ryan Cole
+# kpod-stats "1" "July 2017" "kpod"
+
+## NAME
+kpod-stats - Display a live stream of 1 or more containers' resource usage statistics
+
+## SYNOPSIS
+**kpod** **stats** [*options* [...]] [container]
+
+## DESCRIPTION
+Display a live stream of one or more containers' resource usage statistics
+
+## OPTIONS
+
+**--all, -a**
+
+Show all containers.  Only running containers are shown by default
+
+**--no-stream**
+
+Disable streaming stats and only pull the first result, default setting is false
+
+**--format="TEMPLATE"**
+
+Pretty-print images using a Go template
+
+
+## EXAMPLE
+
+TODO
```

Comment:
As Dan noted, examples would be great.  Although it might seem obvious, talking somewhere about the values in each column would be good too.  As a for instance you've a nice comment down below about how cpu % can get up to 400% on a 4 core machine.  That IMO would be good to document here for that column.